### ISSUE TITLE
No Support for iPhone 5 form factor

### ISSUE THREAD

**Issue by elislade** [iOS10-KIT/UI#17]
Many designers design for 4" form factor and up. It pays off in the end for both the user and designer to design from the smallest form factors. The repository should have two of each Symbol one 4" and one for  4.7".

**Comment by rafaelconde** [comment]
Throw a 5" as well while you're at it. Let me know if you need some screenshots.

**Comment by viewDidAppear** [comment]
I agree. I'm happy to convert some of the stuff over to 4" screen size. @cjmlgrto

**Comment by RamonGilabert** [comment]
That would be really cool, @cocotutch! :)

**Comment by cjmlgrto** [comment]
I might start a branch for a new version of **UI.sketch** using the new resizing features in Sketch 3.9 beta— so the UI is responsive.

**Comment by lanceplaine** [comment]
@cjmlgrto I'm currently working on reorganizing a little bit the views and adding the "responsive" (iPhone SE and iPhone 7 Plus). I personally design on Plus devices.

I just started with the homescreen and plan to do few screens when I have time after work next week.

Currently I have the following naming:
- `homescreen_se` → iPhone SE (320px wide)
- `homescreen` → iPhone 7 (375px wide)
- `homescreen_plus` → iPhone 7 Plus (414px wide)

<img width="1440" alt="screen shot 2016-09-17 at 8 23 54 pm" src="https://cloud.githubusercontent.com/assets/5476205/18612665/b90b78b2-7d14-11e6-9ce3-f9474ed4ac15.png">

(https://github.com/lanceplaine/sketch-apple-ios10/tree/master/mockups/system)

**Should I pull request the clean first or wait to have all of the "system" screens to do a PR?**

Note that if you give me write access to the repo I will be more than happy to spend about 4-5 hours a week helping you maintaining this repo. 

✌️
